saturn-datepicker/src/datepicker/month-view.ts

```diff
@@ -91,6 +91,9 @@ export class SatMonthView<D> implements AfterContentInit {
   /** Whenever user already selected start of dates interval. */
   @Input() beginDateSelected: D | boolean = false;
 
```

Comment:
What if instead of `@Input() beginDateSelected: D | boolean = false;` 
```
@Input() set beginDateSelected(value: D | boolean) { this._beginDateSelected = value } ;
```

Reply:
You mean, instead of setting `this._beginDateSelected` in `_dateSelected` ? It doesn't solve the problem, because this comes from the fact that the value of `beginDateSelected` is set too late.

Reply:
I mean, instead of reading `beginDateSelected`, use only `_beginDateSelected`